python/mxnet/text/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# coding: utf-8
+# pylint: disable=wildcard-import
+"""Text utilities."""
+
+from . import utils
+from .utils import *
+from . import glossary
+from .glossary import *
```

Comment:
Remove `from .glossary import *` unless the classes are intended to exist in `mx.text` namespace.

Reply:
Thanks. I probably will import it because mxnet.text.Embedding, mxnet.text.Glossary looks fine and are consistently used in the documentation.

Reply:
If so, then I think the utils and glossary namespaces are unnecessary. There’s no point in keeping both.

Reply:
resolved.